app/main.py

```diff
@@ -1,3 +1,76 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, distance):
+        if (isinstance(distance, int)):
+            self.km = self.km + distance
+        else:
+            self.km = self.km + distance.km
+        return Distance(self.km)
+
+    def __iadd__(self, distance):
+        if (isinstance(distance, int)):
+            self.km += distance
+        else:
+            self.km += distance.km
+        return self
+
+    def __mul__(self, distance):
+        if (isinstance(distance, int)):
```

Comment:
```suggestion
        if isinstance(distance, int):
```
stick to the PEP8 standard